### ISSUE TITLE
Updater always fails

### ISSUE THREAD

**Issue by Bluscream** [Rem0o/FanControl.Releases#1086]
**Describe the bug**
```
PS D:\Downloads\FanControl> .\Updater.exe
Downloading Fan Control...
An error occured while downloading the update:
An error occurred while sending the request.
Unable to connect to the remote server
An error occured while downloading the update:
An error occurred while sending the request.
Unable to connect to the remote server
Download failed:
```

**Comment by Rem0o** [comment]
Seems like a network issue on your end, otherwise my inbox would be submerged with the same issue.

**Comment by Bluscream** [comment]
I don't see any failure here and i'm not quite sure how to debug it if the updater doesn't give out any information on the issue. i checked wether i can access the bit.ly url and github url from the updater in my browser and they were working just fine there.

**Comment by Rem0o** [comment]
"Unable to connect to the remote server" is the error raised by the HttpClient in .NET. 

Could be many things...

1) firewall blocking
2) vpn/network config blocking
3) no internet

If you do run the updater manually (updater.exe), do you always get the same errror?

**Comment by Bluscream** [comment]
yes, exactly. I tried injecting debug prints with dnSpy but it refuses to recompile

**Comment by Bluscream** [comment]
https://github.com/Aliucord/AliucordManager/pull/21

**Comment by Bluscream** [comment]
Could this be related?
